Remaining jdbc failures are due to ejb jar missing base classes

**Describe the bug**
In the current jdbc run failures:
https://ci.eclipse.org/jakartaee-tck/job/11/job/tck/job/jakarta-jdbc-tck-glassfish/142

com.sun.ts.tests.jdbc.ee.batchUpdate which have missed to update the ejb vehicle jar with the 
		ejb.addClasses(batchUpdateClient.class);

should be:
		ejb.addClasses(batchUpdateClient.class, , ServiceEETest.class, EETest.class);


The majority of the failures are in the com.sun.ts.tests.jdbc.ee.callStmt.* tests where we seem to be missing the stored proceedure jar:
```
Caused by: ERROR 42X51: The class 'com.sun.ts.lib.tests.jdbc.CS_Procs' does not exist or is inaccessible. This can happen if the class is not public or the derby.database.classpath property is missing or incorrectly defined.
	at CommonLibs//org.apache.derby.client.am.ClientStatement.completeSqlca(ClientStatement.java:2116)
	at CommonLibs//org.apache.derby.client.net.NetStatementReply.parsePrepareError(NetStatementReply.java:579)
	at CommonLibs//org.apache.derby.client.net.NetStatementReply.parsePRPSQLSTTreply(NetStatementReply.java:145)
	at CommonLibs//org.apache.derby.client.net.NetStatementReply.readPrepare(NetStatementReply.java:78)
	at CommonLibs//org.apache.derby.client.net.StatementReply.readPrepare(StatementReply.java:64)
	at CommonLibs//org.apache.derby.client.net.NetStatement.readPrepare_(NetStatement.java:201)
	at CommonLibs//org.apache.derby.client.am.ClientStatement.readPrepare(ClientStatement.java:1720)
	at CommonLibs//org.apache.derby.client.am.ClientPreparedStatement.readPrepareDescribeInput(ClientPreparedStatement.java:1927)
	at CommonLibs//org.apache.derby.client.am.ClientPreparedStatement.flowPrepareDescribeInputOutput(ClientPreparedStatement.java:1972)
	at CommonLibs//org.apache.derby.client.am.ClientPreparedStatement.prepare(ClientPreparedStatement.java:315)
	at CommonLibs//org.apache.derby.client.am.ClientConnection.prepareCallX(ClientConnection.java:2012)
	at CommonLibs//org.apache.derby.client.am.ClientConnection.prepareCall(ClientConnection.java:483)
	... 33 more
Caused by: ERROR XJ001: Java exception: 'com.sun.ts.lib.tests.jdbc.CS_Procs: java.lang.ClassNotFoundException'.
	... 45 more
```

The console log shows:
```
[INFO] --- sql:3.0.0:execute (sql-cli) @ glassfish.jdbc-platform-tck ---
[INFO] Executing commands
[ERROR] Failed to execute:  CALL sqlj.install_jar('/home/jenkins/agent/workspace/11/tck/jakarta-jdbc-tck-glassfish/glassfish-runner/platform/jdbc-platform-tck/jdbc-platform-tck-run/target/glassfish8/glassfish/lib/jdbc-platform-tck.jar', 'CTS1.CS_Procs', 0)
[ERROR] java.sql.SQLException: Exception while accessing jar file using URL '/home/jenkins/agent/workspace/11/tck/jakarta-jdbc-tck-glassfish/glassfish-runner/platform/jdbc-platform-tck/jdbc-platform-tck-run/target/glassfish8/glassfish/lib/jdbc-platform-tck.jar'. 
[ERROR] Failed to execute:  CALL SYSCS_UTIL.SYSCS_SET_DATABASE_PROPERTY('derby.database.classpath', 'CTS1.CS_Procs')
[ERROR] java.sql.SQLSyntaxErrorException: The database class path contains an unknown jar file '"CTS1"."CS_PROCS"'.
/sql-cli
```

--- a/glassfish-runner/platform/jdbc-platform-tck/jdbc-platform-tck-run/pom.xml
+++ b/glassfish-runner/platform/jdbc-platform-tck/jdbc-platform-tck-run/pom.xml
@@ -159,15 +159,13 @@
                                     <outputDirectory>${glassfish.lib.dir}</outputDirectory>
                                     <destFileName>tck-porting-lib.jar</destFileName>
                                 </artifactItem>
-                                <!--
                                 <artifactItem>
                                     <groupId>jakarta.tck</groupId>
                                     <artifactId>jdbc-platform-tck</artifactId>
                                     <overWrite>true</overWrite>
-                                    <outputDirectory>${glassfish.lib.dir}</outputDirectory>
+                                    <outputDirectory>${project.build.directory}/jdbc-tck</outputDirectory>
                                     <destFileName>jdbc-platform-tck.jar</destFileName>
                                 </artifactItem>
-                                -->
                                 <artifactItem>
                                     <groupId>jakarta.tck.arquillian</groupId>
                                     <artifactId>arquillian-protocol-lib</artifactId>
@@ -251,7 +249,7 @@
                                 <srcFile>${sql.directory}/${jdbc.db}/${jdbc.db}.ddl.sprocs.sql</srcFile>
                             </srcFiles>
                             <sqlCommand>CREATE SCHEMA CTS1 AUTHORIZATION CTS1;
-CALL sqlj.install_jar('${glassfish.lib.dir}/jdbc-platform-tck.jar', 'CTS1.CS_Procs', 0);
+CALL sqlj.install_jar('${project.build.directory}/jdbc-tck/jdbc-platform-tck.jar', 'CTS1.CS_Procs', 0);
 CALL SYSCS_UTIL.SYSCS_SET_DATABASE_PROPERTY('derby.database.classpath', 'CTS1.CS_Procs');</sqlCommand>
                         </configuration>
                     </execution>

--- a/tcks/profiles/platform/jdbc/src/main/java/com/sun/ts/tests/jdbc/ee/batchUpdate/batchUpdateClientEJB.java
+++ b/tcks/profiles/platform/jdbc/src/main/java/com/sun/ts/tests/jdbc/ee/batchUpdate/batchUpdateClientEJB.java
@@ -84,7 +84,7 @@ public class batchUpdateClientEJB extends batchUpdateClient implements Serializa
 		ejb.addPackages(false, "com.sun.ts.tests.common.vehicle");
 		ejb.addPackages(false, "com.sun.ts.tests.common.vehicle.ejb");
 		ejb.addPackages(true, "com.sun.ts.lib.harness");
-		ejb.addClasses(batchUpdateClient.class);
+		ejb.addClasses(batchUpdateClient.class, ServiceEETest.class, EETest.class);
 
 		resURL = batchUpdateClientEJB.class
 				.getResource("/com/sun/ts/tests/jdbc/ee/batchUpdate/batchUpdate_ejb_vehicle_ejb.jar.sun-ejb-jar.xml");

